specification/components/schemas/provisioningCertificateChainData.v1.json

```diff
@@ -5,9 +5,9 @@
     "x-examples": {
       "example": {
         "provisioningCertificateChain": {
-          "subCA1Certificate": "",
-          "subCA2Certificate": "",
-          "certificate": ""
+          "subCA1Certificate": "MIICQDCCAeWgAwIBAgICBNMwCgYIKoZIzj0EAwIwRDELMAkGA1UEBhMCREUxDTALB",
```

Comment:
These base64 does not correspond to real certificate (truncated ?). Maybe we can put as example real "test certificates"

Reply:
quiet long - I can add if desired